config/settings.yml

```
@@ -0,0 +1,5 @@
+user:
+  name:
+    max_length: 150
+  email:
+    max_length: 255
```

```suggestion
length:  
  digit_150: 150
  digit_250: 255
```

ri đểtái sử dụng dc tốt nhé